<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_i_inventory_stocking_change_Dialog</class>
 <widget class="QDialog" name="w_i_inventory_stocking_change_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>199</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="lds_toolbar" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="lds_frame_pixmap" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="lds_toolbar" name="widget_2" native="true"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>单据号码</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ch_sheetno"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>责任人</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="lds_ComboBox" name="comboBox_vch_handle">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>备注说明</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="7">
         <widget class="QLineEdit" name="lineEdit_vch_memo"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>制单人</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_vch_operid"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>制单日期</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEdit_dt_operdate"/>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>审核人</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLineEdit" name="lineEdit_vch_audit"/>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>审核日期</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QLineEdit" name="lineEdit_dt_audit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="lds_tableview_sum" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_tableview_sum</class>
   <extends>QTableView</extends>
   <header>lds_tableview_sum.h</header>
  </customwidget>
  <customwidget>
   <class>lds_frame_pixmap</class>
   <extends>QFrame</extends>
   <header>lds_frame_pixmap.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
